core/trino-parser/src/main/java/io/trino/sql/tree/DefaultTraversalVisitor.java

```diff
@@ -758,6 +758,16 @@ protected Void visitAddColumn(AddColumn node, C context)
         return null;
     }
 
+    @Override
+    protected Void visitCreateSchema(CreateSchema node, C context)
+    {
+        for (Property property : node.getProperties()) {
+            process(property, context);
+        }
+
```

Comment:
should this visit `principal` too?

(i think it shouldn't)

cc @martint

Reply:
I don't think so, since it's an identifier, not an expression.